mmv1/third_party/terraform/resources/resource_compute_instance_from_machine_image.go.erb

```diff
@@ -120,12 +120,28 @@ func resourceComputeInstanceFromMachineImageCreate(d *schema.ResourceData, meta
 		return err
 	}
 
-	tpl, err := ParseMachineImageFieldValue(d.Get("source_machine_image").(string), d, config)
+	// ServiceAccount is required when the image is from a different project
+	// it will be overridden by a call, SetServiceAccount(), if service_account is provided in the config
+	accounts := make([]*computeBeta.ServiceAccount, 1)
+	accounts[0] = &computeBeta.ServiceAccount{
+		Email:  "default",
+		Scopes: nil,
+	}
+	instance.ServiceAccounts = accounts
```

Comment:
Doesn't this need to be _before_ `expandComputeInstance` to be overriden by the `service_account` field?

At the very least, should we check to see if `instance.ServiceAccounts` is empty before overwriting it here?